plugins/by-name/lsp_signature/default.nix

```
@@ -0,0 +1,206 @@
+{ lib, helpers, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "lsp_signature";
```

```suggestion
  name = "lsp-signature";
```
Both the module and source files should be named `lsp-signatures`.